sonata-project/page-bundle/4.0/src/Entity/SonataPagePage.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Sonata\PageBundle\Entity\BasePage;
+
+/**
+ * @ORM\Entity
+ * @ORM\Table(name="page__page")
+ * @ORM\HasLifecycleCallbacks
```

Comment:
I don't think adding this is necessary. Did you find any problem without it?

Reply:
I don't remember having this line, and the life cycle callbacks were executed anyway.

Reply:
the only thing that I found that probably use it, is: https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/Entity/BaseBlock.php#L21

But it works without `HasLifecycleCallbacks`

probably those line make this prePersist and update works: https://github.com/sonata-project/SonataPageBundle/blob/4.x/src/Resources/config/doctrine/BaseBlock.orm.xml#L11-L14


Then ok I'll remove it ;)